src/error.rs

```diff
@@ -15,3 +17,19 @@ pub enum Error<Espi> {
     /// Packet exceeds maximum size (255 for send_large)
     PacketTooLarge,
 }
+
+impl<Espi: Display> Display for Error<Espi> {
```

Comment:
Display wasn't implemented for a reason, that's it significantly increases flash size as the formatting is pretty heavy weight (or at least it was last time when I was dealing with those issues).

Reply:
Using `Display` will increase code size, sure. But just having it implemented shouldn't be a problem, the code shouldn't be included in the binary if it is never called.

Reply:
It's true that the compiler should be smart enough to remove it when it's not referenced.